pages/video-detail-edit/video-detail-edit.view.html

```diff
@@ -0,0 +1,74 @@
+<!--<topbar></topbar>-->
+
+<md-toolbar  md-scroll-shrink="true" layout="row" flex >
+	<div class="md-toolbar-tools">
+		<!--<span class="mobile-menu">-->
+			<!--<img src="/img/menu.svg" alt="img_menu">-->
+		<!--</span>-->
+		<span class="logo">
+			<img ui-sref="gallery" src="/img/brand/logo_low.png" alt="logo">
+		</span>
+		<h2 md-truncate flex></h2>
+
+		<md-icon ui-sref="upload" md-svg-src="/img/upload-arrow.svg" class="upload-drop-icon" aria-label="Upload"></md-icon>
+		<md-icon md-svg-src="/img/search.svg" class="search-drop-icon" aria-label="Search"></md-icon>
+	</div>
+</md-toolbar>
+
+<div ng-controller="VideoDetailEditController as EditCtrl" class="video-edit-page" layout flex layout-fill>
+	<error404 ng-if="!EditCtrl.video && !EditCtrl.loading" title="{{'VIDEO_ERROR' | translate}}" message="{{'VIDEO_NOT_FOUND' | translate}}"></error404>
+
+	<loading-card ng-if="!EditCtrl.video && EditCtrl.loading"></loading-card>
+	<div ng-if="!EditCtrl.loading && EditCtrl.video" flex-gt-md="80" flex-offset-gt-md="10" class="video-edit-page__main-details">
+		<form name="videoForm" novalidate ng-submit="videoForm.$valid && EditCtrl.update()" layout="row" layout-wrap flex="100">
+			<div flex-gt-sm="60" flex="100">
+				<ng-include src="'pages/video-detail-edit/partials/main-player.html'"></ng-include>
+			</div>
+
+			<div flex-gt-sm="40" flex="100">
+				<ng-include src="'pages/video-detail-edit/partials/basic-details.html'"></ng-include>
+			</div>
+
+			<div flex="100">
+				<ng-include src="'pages/video-detail-edit/partials/aditional-details.html'"></ng-include>
+			</div>
+
+			<div flex="100">
+				<ng-include src="'pages/video-detail-edit/partials/video-file.html'"></ng-include>
+			</div>
+
+			<div flex="100">
+				<ng-include src="'pages/video-detail-edit/partials/video-map.html'"></ng-include>
```

Comment:
Even when it works, ng-include is known for make the digest cycles slow, and for cause so much bugs in production for forgiven things inside templates, or for parent scope's unexpected changes. Instead of that, there are recomended to create directives (or components), and pass as a parameter whatever it need inside.

This thing is not much mor performant, but cleaner, since this new directives are totally individual, and don't depend of a parent $scope controller to exist, but it might be passed to its own scope, wich make it very easy to mantain. Also, as it creates a new scope, there's not danger for inner stuff to be changed from outside this template, and enable a way to link information which shall be shared (in this case, the controller instance, or its props/params)

[Some literature about this](http://www.codelord.net/2016/11/17/avoiding-ng-include-for-elegenace-and-performance/). Not a real thing to change now, but just to keep in mind ;)

Reply:
ok, we'll investigate further. I've already had this in mind since several partials (as map) could be used in other places (such as video detail page). This is a fast and working first pass/approach that simplefies and better structure a complex view like this by now